test/e2e-go/e2e_go_suite_test.go

```diff
@@ -48,9 +45,17 @@ var _ = BeforeSuite(func() {
 	By("creating a new test context")
 	tc, err = testutils.NewTestContext(testutils.BinaryName, "GO111MODULE=on")
 	Expect(err).NotTo(HaveOccurred())
-
-	By("creating a new directory")
-	Expect(tc.Prepare()).To(Succeed())
+	
+	tc.Domain = "example.com"
+	tc.Group = "cache"
+	tc.Version = "v1alpha1"
+	tc.Kind = "Memcached"
+	tc.Resources = "memcacheds"
+	tc.ProjectName = "memcached-operator"
+	tc.Kubectl.Namespace = fmt.Sprintf("%s-system", tc.ProjectName)
+
+	By("copying sample to a temporary e2e directory")
+	Expect(exec.Command("cp", "-r", "../../testdata/go/memcached-operator", tc.Dir).Run()).To(Succeed())
 
```

Comment:
```suggestion
	_, err := tc.Run(exec.Command("cp", "-r", "../../testdata/go/memcached-operator", "."))
	Expect(err).NotTo(HaveOccurred())

```

Reply:
The existing cp command effectively creates e2e-agfv and copies the contents of ./testdata/... into e2e-<abcd> then, if we follow up your suggestion here it will not happens.

Reply:
`TestContext.Run()` runs the command in `TestContext.Dir`, which will be `test/e2e-go/e2e-XXXX`.

Reply:
Note that we are not calling anuy longer in the e2e tests 

```
By("creating a new directory")
Expect(tc.Prepare()).To(Succeed())
```

Then, before to run the cp the `e2e-abcd` dir does not exist. So, if we follow up your suggestion here we will face the following error: 

```
Unexpected error:
      <*errors.errorString | 0xc0001c9a20>: {
          s: "cp -r ../../testdata/go/memcached-operator . failed with error: (chdir /Users/camilamacedo/go/src/github.com/operator-framework/operator-sdk/test/e2e-go/e2e-uirp: no such file or directory) ",
      }
      cp -r ../../testdata/go/memcached-operator . failed with error: (chdir /Users/camilamacedo/go/src/github.com/operator-framework/operator-sdk/test/e2e-go/e2e-uirp: no such file or directory) 
  occurred

```